### ISSUE TITLE
Converting Markdown string to HTML with emojis

### ISSUE THREAD

**Issue by Ivan-Cantarino** [iwasrobbed/Down#60]
*[x] I've searched other open/closed issues for duplicates before opening up this new issue.

# Report

## What did you do?

I've inserted a markdown string with some emojis into it and used `Down(markdownString: str).toHTML()`

The markdown string was something like: 
```
markdownString = """
		# Header 01
		## header 02
		
		**some bold text 0**
		*some italic text 0*
		
		> Blockquote text
		> multilined
		
               text with emojis: 👍😀🔞😈🤢🤖
		"""
```
## What did you expect to happen?

Expected the `toHTML()` function parsed the emojis, so it would appear inside the `WebView`.

## What happened instead?

The emojis unicode weren't parsed as expected, instead, for the emojis inserted (see above) the output inside the `WebView` after using the `toHTML()` function was the following:
(I'm reducing the unnecessary text)

``` 
text with emojis: ðŸ‘ðŸ˜€ðŸ”žðŸ˜ˆðŸ¤¢ðŸ¤–
```

**Comment by 128keaton** [comment]
Okay--I did a bit of investigating on this issue. It looked like the header wasn't being set properly, but I had to make sure cmark wasn't screwing up the output. First, I built cmark from the latest source and ran your test file through it. It outputted properly in my terminal, but spitting out to raw HTML rendered it exactly like your example. I simply pre-pended this to the html file: 
`<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />`
